controllers/statusmanager/status_manager.go

```diff
@@ -292,13 +295,13 @@ func (status *StatusManager) CombineConditions(conditions *[]configv1.ClusterOpe
 	for _, newCondition := range *newConditions {
 		existingCondition := v1helpers.FindStatusCondition(*conditions, newCondition.Type)
 		if existingCondition == nil {
-			v1helpers.SetStatusCondition(conditions, newCondition)
+			v1helpers.SetStatusCondition(conditions, newCondition, clock.RealClock{})
 			messages += fmt.Sprintf("%v. ", newCondition)
 			changed = true
 		} else if existingCondition.Status != newCondition.Status ||
 			existingCondition.Reason != newCondition.Reason ||
 			existingCondition.Message != newCondition.Message {
-			v1helpers.SetStatusCondition(conditions, newCondition)
+			v1helpers.SetStatusCondition(conditions, newCondition, clock.RealClock{})
```

Comment:
Are changes like this needed due to the bump in library-go?

Reply:
Yes